app/controllers/vignettes/slides_controller.rb

```diff
@@ -0,0 +1,72 @@
+module Vignettes
+  class SlidesController < ApplicationController
+    before_action :set_questionnaire
```

Comment:
I think now is the time for your TODO note: "Limit feature to create Questionnaire to certain users". I.e. authorize all controller endpoints (see other MaMpf controllers for inspiration).